contentcuration/contentcuration/frontend/channelEdit/components/Clipboard/index.vue

```diff
@@ -11,6 +11,7 @@
       clipped
       class="clipboard elevation-4"
       v-bind="$attrs"
+      style="z-index: 5;"
```

Comment:
The z-index for the drawer should be 16 per the Material guidelines.

![material-elevations](https://user-images.githubusercontent.com/6356129/97916420-cd396780-1d07-11eb-85c0-4b8e1e7776eb.png)

Reply:
Unfortunately, vuetify's overlay has the z-index at 5, which causes this to happen 
![image](https://user-images.githubusercontent.com/7447496/98043750-9aac6f00-1dda-11eb-8467-6c9afa5cbb3b.png)

I can set the clipboard/tree navigation's z-index to 4 instead, but I'd rather not mess too much with the overlay z-index as it's used in a lot of different areas